npm-shrinkwrap.json

```diff
@@ -5970,7 +5970,7 @@
       "dependencies": {
         "camelcase-keys": {
           "version": "2.1.0",
-          "resolved": "https://registry.npmjs.org/camelcase-keys/-/camelcase-keys-2.1.0.tgz",
+          "resolved": "http://registry.npmjs.org/camelcase-keys/-/camelcase-keys-2.1.0.tgz",
```

Comment:
Why is there a diff here? Seems unnecessary.

Reply:
Correct, my bad